docs/configuration.adoc

```diff
@@ -286,3 +288,8 @@ image_signing_keys = ["ocid1.key.oc1....", "ocid1.key.oc1...."]
 ----
 {uri-terraform-options}#kms-integration[Reference]
 
+== Configure defined tags
+
+The {uri-oci-tags-overview}#[defined tags] *must* first be manually created before using them as input parameters. 
+If a tag is a list type, you must specify one of the predefined options when applying it. 
```

Comment:
Can you please give an example for both? Your example from `terraform.tfvars.example` should be good